source/extensions/filters/http/ext_proc/BUILD

```diff
@@ -141,3 +142,14 @@ envoy_cc_library(
         "@envoy_api//envoy/service/ext_proc/v3:pkg_cc_proto",
     ],
 )
+
+envoy_cc_library(
+    name = "on_receive_message_decorator",
```

Comment:
As this is an interface, I suggest adding the "_interface" suffix to the build target name (consistent with other interfaces).